app/assets/stylesheets/_country-notice.scss

```diff
@@ -0,0 +1,12 @@
+.country-notice {
+  background-color: govuk-colour('blue');
+  padding: govuk-spacing(4) govuk-spacing(4) govuk-spacing(1);
+
+  @include govuk-media-query($from: tablet) {
+    padding: govuk-spacing(5) govuk-spacing(5) govuk-spacing(1);
+  }
+}
+
+.country-notice__text {
+  color: govuk-colour('white');
+}
```

Comment:
If you want this for Production you'll need to add this to Frontend for transaction pages.